__tests__/database/schema/ChatSchema.test.ts

```diff
@@ -11,6 +11,7 @@ const chatSchema = new ChatSchema({
   createdAt: 'createdAt',
   updatedAt: 'updatedAt',
   rawJson: {},
+  attachmentJson: [],
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachmentJson` field is initialized as an array, but in the database it should be a string that contains serialized JSON. This could lead to inconsistencies when testing.
```diff
-  attachmentJson: [],
+  attachmentJson: '[]',
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
It's correct because this is not string. String only when we saving to sqlite